app/models/mixins/miq_web_server_runner_mixin.rb

```diff
@@ -14,4 +14,32 @@ def do_before_work_loop
     # Since puma/thin traps interrupts, log that we're going away and update our worker row
     at_exit { do_exit("Exit request received.") }
   end
+
+  module ClassMethods
+    def start_worker(*args)
+      worker = self.new(*args)
+      _log.info("URI: #{worker.worker.uri}")
+
+      # Do all the SQL worker preparation in the main thread
+      worker.prepare
+
+      # TODO: Need to rename build_command_line to options_hash or something
+      options = corresponding_model.build_command_line(:Port => args.first[:Port])
```

Comment:
TODO: Rename this method and drop the command line column since we're not building a command line with fork.